src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -12,17 +12,17 @@
 using System.Diagnostics;
 using System.IO;
 using System.Text;
-using System.Runtime.InteropServices;
 using System.Runtime.CompilerServices;
 using System.Runtime.ExceptionServices;
+using System.Runtime.InteropServices;
 
 #if SUPPORT_JIT
 using Internal.Runtime.CompilerServices;
 #endif
 
-using Internal.TypeSystem;
-
 using Internal.IL;
+using Internal.JitInterface;
```

Comment:
Why do we need this?

Reply:
Hmm, probably just some copy & paste bug, will remove.

Reply:
Fixed in 2nd commit.